core/server/proxy/src/main/java/alluxio/proxy/s3/S3AuditContext.java

```diff
@@ -28,6 +28,7 @@ public class S3AuditContext implements AuditContext {
   private String mObject;
   private long mCreationTimeNs;
   private long mExecutionTimeNs;
+  private String mComplement;
```

Comment:
Can this be something like `List<String> mMetadata` so that you can include arbitrary amounts of metadata?

Reply:
done.